src/plugins/connectionPlugin.ts

```diff
@@ -336,11 +336,19 @@ export const connectionPlugin = (
 
   // Define the plugin with the appropriate configuration.
 
+  console.log(
+    printedGenTypingImport({
+      module: getOwnPackage().name,
+      bindings: ["core", "connectionPluginCore"],
+    })
+  );
+
```

Comment:
I know it's wip, just a small reminder to not forget to remove that console.log 🙈